src/MonoMod.SourceGen.Internal/Utils/FastDelegateInvokersGenerator.cs

```diff
@@ -79,6 +79,7 @@ private sealed record GeneratorMethod(IMethodSymbol Method, SyntaxTokenList Modi
             ctx.AddSource($"{methodName}.g.cs", sb.ToString());
         }
 
+#pragma warning disable CA1305 // Specify IFormatProvider
```

Comment:
As mentioned before, I'd prefer `SuppressMessage` attributes, or in this case, explicitly passing `null` to the methods which take it.